docs/src/components/landing/Hero.js

```diff
@@ -38,44 +128,97 @@ export default function Hero() {
           </Box>
           <Chip label="Beta" component="span" color="primary" size="small" variant="outlined" />
         </Typography>
-        <Typography variant="h1" sx={{ my: 2 }}>
-          Low-code
+        <Typography variant="h1" sx={{ my: 1, minWidth: { xs: 'auto', sm: 600 } }}>
+          Turn your <TypingAnimation />
           <br />
-          <GradientText>admin builder</GradientText>
+          <GradientText>into UIs</GradientText>
         </Typography>
-        <Typography color="text.secondary" sx={{ maxWidth: 520, mx: 'auto', mb: 4 }}>
-          Build scalable and secure internal tools locally. Use your own IDE, drag and drop
+        <Typography color="text.secondary" sx={{ maxWidth: 520, mb: 2, textWrap: 'balance' }}>
+          Build scalable and secure internal tools locally. Use your own IDE, <br /> drag and drop
           pre-built components or create your own.
         </Typography>
-        <GetStartedButtons
-          sx={{ justifyContent: 'center' }}
-          installation={'npx create-toolpad-app'}
-          to={ROUTES.toolpadQuickstart}
-        />
+        <GetStartedButtons installation={'npx create-toolpad-app'} to={ROUTES.toolpadQuickstart} />
         <Box
           sx={{
-            mx: 'auto',
-            mt: 6,
+            mt: 2,
             display: 'flex',
-            justifyContent: 'center',
             alignItems: 'center',
-            gap: 4,
+            justifyContent: { xs: 'space-around', sm: 'start' },
+            gap: 2,
           }}
         >
-          <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+          <GithubStars />
+          <Box sx={{ display: 'flex', alignItems: 'center', gap: 0.8 }}>
             <SvgMuiLogo width={20} />
-            <Typography color="text.primary" fontWeight="medium" variant="body2">
-              Powered by MUI
+            <Typography color="text.secondary" fontWeight="medium" variant="body2">
+              Powered by Material UI
             </Typography>
           </Box>
-          <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
-            <DataObjectRoundedIcon color="primary" />
-            <Typography color="text.primary" fontWeight="medium" variant="body2">
+          <Box sx={{ display: 'flex', alignItems: 'center', gap: 0.8 }}>
+            <KeyRoundedIcon color="primary" />
+            <Typography color="text.secondary" fontWeight="medium" variant="body2">
               Open source
```

Comment:
Perhaps we could replace this "Open source" with the GitHub button? I mean, one sort of implies the other